source/nodejs/adaptivecards-fabric/src/components/inputs/InputChoiceSetFabric.tsx

```diff
@@ -20,9 +20,8 @@ export class InputChoiceSetFabric extends Shared.ReactInputElement {
     private selectedValues: string[] = [];
 
     public parse = (json: any, errors?: AC.IValidationError[]) => {
-        this.id = json.id ? AC.getStringValue(json.id) : null;
```

Comment:
You do not appear to be parsing/handling the `wrap` and `placeholder`  properties.

Reply:
wrap is 1.2 feature which is not supported by this extension.